src/assets/wise5/components/aiChat/aiChatMessage.ts

```diff
@@ -0,0 +1,4 @@
+export interface AiChatMessage {
+  content: string;
+  role: string;
```

Comment:
Specify available options using literal types?
```suggestion
  role: 'assistant' | 'system' | 'user';
```